content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -585,24 +585,24 @@ $ curl "$INGRESS_HOST:$INGRESS_PORT/ip" -s -o /dev/null -w "%{http_code}\n"
 
 1. Remove authentication policy:
 
-    {{< text bash >}}
-    $ kubectl -n istio-system delete requestauthentication jwt-example
-    {{< /text >}}
+{{< text bash >}}
```

Comment:
Removing the indents here have now caused the numbering to reset, so they all show as 1. in the deployed page (see https://deploy-preview-9419--preliminary-istio.netlify.app/latest/docs/tasks/security/authentication/authn-policy/#cleanup-part-3). The netlify job details will point to a generated website so you can view what you changes actually look like.

Reply:
I rechecked the code and then use `Make Gen` and `make lint` to check it. It should be all right now.